server/util/alert/alert.go

```diff
@@ -11,6 +14,23 @@ import (
 // `name` is used as a metric label and thus should be a constant string. Do not include dynamic information.
 //
 // Use this for cases where it's worth knowing about the event, but not worth introducing a standalone metric & alert.
-func UnexpectedEvent(name string) {
+//
+// Example usages:
+// 	alert.UnexpectedEvent("cannot_unmarshal_proto")
+//	alert.UnexpectedEvent("cannot_unmarshal_proto", "invocation_id %s err: %s", invocation_id, err)
```

Comment:
i thought we wanted no dynamic information in here?

Reply:
Only in the name. The debug message can contain additional information useful for debugging. It is not included in the exported metric.